LayoutTests/fast/webgpu/fuzz-149183515-reduced.html

```diff
@@ -0,0 +1,163 @@
+    });
+    let pipeline1 = device0.createComputePipeline({
+        layout: 'auto',
+        compute: {
+            module: shaderModule1,
+            constants: {
+            }
+        }
+    });
+    let externalTexture3 = device0.importExternalTexture({
+        source: videoFrame1,
+        colorSpace: 'srgb'
+    });
+    let autogeneratedBindGroupLayout1 = pipeline1.getBindGroupLayout(1);
+    let commandEncoder2 = device0.createCommandEncoder();
+    let computePassEncoder8 = commandEncoder2.beginComputePass();
+    computePassEncoder8.setPipeline(pipeline1);
+    let buffer17 = device0.createBuffer({
+        size: 60,
+        usage: GPUBufferUsage.COPY_DST | GPUBufferUsage.STORAGE
+    });
+    let autogeneratedBindGroupLayout5 = pipeline1.getBindGroupLayout(0);
+    let bindGroup17 = device0.createBindGroup({
+        layout: autogeneratedBindGroupLayout1,
+        entries: [{
+            binding: 332,
+            resource: {
+                buffer: buffer17
+            }
+        }, {
+            binding: 150,
+            resource: externalTexture3
+        }],
+    });
+    let bindGroup18 = device0.createBindGroup({
+        layout: autogeneratedBindGroupLayout5,
+        entries: [{
+            binding: 150,
+            resource: externalTexture3
+        }, {
+            binding: 332,
+            resource: {
+                buffer: buffer7,
+                size: 48
+            }
+        }],
+    });
+    computePassEncoder8.setBindGroup(0, bindGroup17, new Uint32Array(1214), 774, 0);
+    computePassEncoder8.setBindGroup(1, bindGroup18);
+    computePassEncoder8.dispatchWorkgroups(2);
+    computePassEncoder8.end();
+    let commandBuffer1 = commandEncoder2.finish();
+    device0.queue.submit([commandBuffer1]);
+    await device0.queue.onSubmittedWorkDone();
+}
+
+onload = async () => {
+  try {
+  let promises = [ window0() ];
+  console.log('promises created');
+  let results = await Promise.allSettled(promises);
+  for (let result of results) {
+    if (result.status === 'rejected') { throw result.reason; }
+  }
+  console.log('the end')
+  console.log(location);
+  } catch (e) {
+    console.log('error');
+    console.log(e);
+    console.log(e[Symbol.toStringTag]);
+    console.log(e.stack);
+    if (e instanceof GPUPipelineError) {
+      console.log(`${e} - ${e.reason}`);
+      
+    } else if (e instanceof DOMException) {
+      if (e.name === 'OperationError') {
+      console.log(e.message);
+      
+      } else if (e.name === 'InvalidStateError') {
+      } else {
+        console.log(e);
+        
+      }
+    } else if (e instanceof GPUValidationError) {
+      
+    } else if (e instanceof GPUOutOfMemoryError) {
+      
+    } else if (e instanceof TypeError) {
+      console.log(e);
+      
+    } else {
+      console.log('unexpected error type');
+      console.log(e);
+      
+    }
+  }
+  globalThis.testRunner?.notifyDone();
```

Comment:
may need to move `globalThis.testRunner?.dumpAsText();` right above `notifyDone`